pkg/cmd/addcmd.go

```diff
@@ -53,6 +53,8 @@ func (c *Config) newAddCmd() *cobra.Command {
 	flags.BoolVarP(&c.Add.template, "template", "T", c.Add.template, "Add files as templates")
 	flags.BoolVar(&c.Add.TemplateSymlinks, "template-symlinks", c.Add.TemplateSymlinks, "Add symlinks with target in source or home dirs as templates") //nolint:lll
 
+	flags.MarkDeprecated("autotemplate", "it will be removed in a future release")
```

Comment:
This needs a quick

```go
    if err := flags.MarkDeprecated(...); err != nil {
        panic(err)
    }
```

so we get an error if we remove the flag and forget to remove the `flag.MarkDeprecated`.